packages/scss/src/components/readMore/component.scss

```diff
@@ -0,0 +1,54 @@
+@use '@lucca-front/scss/src/commons/utils/namespace';
+
+@use '@lucca-front/scss/src/commons/utils/a11y';
+
+@mixin component($atRoot: namespace.$defaultAtRoot) {
+	position: relative;
+	display: block;
+
+	@at-root ($atRoot) {
+		.readMore-link.link {
+			position: absolute;
+			inset: auto 0 var(--pr-t-spacings-50) auto;
+			display: var(--components-readMore-link-display);
+			flex-direction: column;
+			z-index: 1;
+			line-height: var(--sizes-M-lineHeight);
+			background-image: linear-gradient(
+				to right,
+				transparent,
+				var(--components-readMore-link-backgroudColor) calc(var(--components-readMore-link-paddingInlineStart) - var(--pr-t-spacings-100))
+			);
+			padding-inline-start: var(--components-readMore-link-paddingInlineStart);
+			font-weight: 600;
+			text-decoration: none;
+		}
+
+		.readMore-textFlow {
+			display: -webkit-box;
+			-webkit-line-clamp: var(--components-readMore-lineClamp);
+			-webkit-box-orient: vertical;
+			overflow: hidden;
+			padding: var(--pr-t-spacings-50);
+			margin: calc(var(--pr-t-spacings-50) * -1);
+
+			> p {
+				&:first-child {
+					margin-block-start: 0;
```

Comment:
Ok, got it, Text Flow is actually called in Read More component, but: 
- It doesn't automatically import Read More component
- Imho, it should be a product choice to use or note Text Flow. It seems overkill to apply by default in cases like table cell content, single comment, etc. By letting it in team hands, we would clean a bunch of CSS lines (to discuss!)

Reply:
- Did you mean that I wasn't automatically loading TextFlow? If that's what you mean, it's been corrected.
- The margin-trim (which I do manually for lack of support) ensures that there are no annoying margins, and I find it simplifies the component. (You can tell me more this afternoon if you need to.)